com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -3729,6 +3743,43 @@ private void ReplaceFile ()
             }
         }
 
+        private string SaveMetafile()
+        {
+            // create temp file for original metafile
+            try {
+                var tempMetafilePath = Path.GetTempFileName();
+                
+                // get relative path for original fbx's metafile
+                var metafile = VersionControl.Provider.GetAssetByPath("Assets" + m_lastFilePath.Substring(Application.dataPath.Length)).metaPath;
+            
+                // save it to a temp file
+                try {
+                    File.Copy(metafile, tempMetafilePath, true);
+                } catch(IOException) {
+                    Debug.LogWarning (string.Format("Failed to copy file {0} to {1}", metafile, tempMetafilePath));
```

Comment:
Please add a little more infor, i.e. "The import settings for the FBX asset will be reset to the defaults"

Reply:
Here you should return ""